lib/default_tracking.dart

```diff
@@ -0,0 +1,77 @@
+/// Tracking default events.
+/// 
+/// Different platforms have different default events. 
+/// Refer to platform specific docs for more details.
+/// 
+/// * [iOS](https://www.docs.developers.amplitude.com/data/sdks/ios-swift/#tracking-default-events)
+/// * [Android](https://www.docs.developers.amplitude.com/data/sdks/android-kotlin/#tracking-default-events)
+/// * [Web](https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-default-events)
+
+class DefaultTrackingOptions {
+  // Available on all platforms: iOS, Android, and Web
+  final bool sessions;
+  // Available on iOS and Android
+  final bool appLifecycles;
+  final bool screenViews;
+  // Available on Android
+  final bool deepLins;
```

Comment:
typo

Reply:
Fixed in https://github.com/amplitude/Amplitude-Flutter/pull/162/commits/fbbb383c7389108ff90dfb517eac5d7003cd3ca1